app/infrastructure/broadcaster.py

```
@@ -1,44 +1,41 @@
-import asyncio
-import contextlib
-from typing import Optional
+import orjson
+import logging
+from typing import AsyncGenerator, Optional
```

Ruff (0.14.7)

Import from `collections.abc` instead: `AsyncGenerator`

Import from `collections.abc`

(UP035)